.terraform-version

```diff
@@ -1 +1 @@
-0.11.14
+0.12.13
```

Comment:
Since we are effectively on 0.12 everywhere, we can get rid of this file.

Reply:
Other people might not be though? Since this will be a terraform 0.12 branch (or rather the old commit will be the terraform 0.11 branch) shouldn't we leave this in?

Reply:
I think you can kill it. The problem is when you're on 0.12.18 and this this is 0.12.13 you have to downgrade your computer or update this. I think I added this file and it can go.